apps/src/lab2/views/dialogs/StartOverDialog.tsx

```diff
@@ -0,0 +1,42 @@
+import Typography from '@cdo/apps/componentLibrary/typography';
+import React from 'react';
+import {BaseDialogProps} from './DialogManager';
+import moduleStyles from './start-over-dialog.module.scss';
+const commonI18n = require('@cdo/locale');
+
+/**
+ * Start Over dialog used in Lab2 labs.
+ */
+const StartOverDialog: React.FunctionComponent<BaseDialogProps> = ({
+  handleConfirm,
+  handleCancel,
+}) => {
+  return (
+    <div className={moduleStyles.startOverDialog}>
+      <Typography semanticTag="h1" visualAppearance="heading-lg">
```

Comment:
@levadadenys I know you're out for a while so this may be merged before you get back, but tagging you in case you have any feedback on how we're using the Typography elements here. Feel free to comment even after this is merged.